Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/BaseMethodDeclaration.cs

```diff
@@ -74,14 +74,14 @@ public static Doc Print(CSharpSyntaxNode node)
 
             var docs = new List<Doc> { ExtraNewLines.Print(node) };
 
-            if (attributeLists.HasValue)
+            if (attributeLists.HasValue && attributeLists.Value.Any())
```

Comment:
You could do both these checks in a single pattern match, if you think it looks better:
```
attributeLists is { HasValue: true, Value: { Count: > 0 } }
```